dora/core/server/worker/src/main/java/alluxio/worker/dora/PagedDoraWorker.java

```diff
@@ -207,13 +207,10 @@ protected PagedDoraWorker(
   }
 
   @VisibleForTesting
-  protected UnderFileSystem getUfsInstance(String ufsUriStr) {
+  public UnderFileSystem getUfsInstance(String ufsUriStr) {
```

Comment:
allow objects holding the worker ref to getUfsInstance. This helps our next step dep injection refactors.